gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc

```diff
@@ -0,0 +1,24 @@
+= Viewing Logs in Runtime Manager
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: runtime-manager::flex-gateway-monitoring.adoc, flex-view-logs-in-monitoring.adoc
+:imagesdir: ../assets/images
+
+Use Runtime Manager to view the runtime logs of your Flex Gateway. Runtime Manager enables you to search for keywords within the logs and to filter by *Replica*, *Time Range*, and *Log Level*. 
```

Comment:
I would add the same warning that we have on the access logs page, that states the storage limits of 100mb or 30 days

Reply:
here https://docs.mulesoft.com/gateway/latest/flex-conn-message-log